### ISSUE TITLE
Bug: "Unterminated regular expression" when parsing division inside a computed property

### ISSUE THREAD

**Issue by axefrog** [cherow/cherow#333]
**Code**

```js
a = { ['foo']: n / 1 }
```

https://cherow.github.io/cherow/?code=a%20%3D%20%7B%20%5B'foo'%5D%3A%20n%20%2F%201%20%7D&method=parse&range=false&loc=undefined&next=false&module=false&raw=false&jsx=undefined&directives=undefined&attachComment=undefined&webCompat=false

**Expected behavior:**

Parses correctly.

**Actual behavior:**

```
Unterminated regular expression (2:31)
  ['foo']: n / 1
```

**Comment by axefrog** [comment]
@KFlash Thanks for fixing that! :)

**Comment by axefrog** [comment]
@KFlash Not sure if I'm misreading, but it appears the fix was applied to the 1.6.x codebase. In my project I'm periodically just manually grabbing a copy of `cherow.esm.js` from the `dist` folder in the dev branch.

**Comment by KFlash** [comment]
@axefrog As you know I'm retired, but I spent some time porting my private parser code to open source.  If you run into cases like this or cases where you are not sure they should pass / fail you can validate with [meriyah](https://github.com/meriyah/meriyah)
